Documentation/Conditions/Index.rst

```diff
@@ -1229,6 +1229,16 @@ site()
             # ...
         [END]
 
+    Site settings can also be used in the conditions in TypoScript constants:
+
+    ..  code-block:: typoscript
+        :caption: EXT:site_package/Configuration/TypoScript/constants.typoscript
+ 
+        my.constant = myValue
+        [traverse(site('configuration'), 'settings/some/setting') == 'someValue']
+          # This works but is rather ugly to rely on
+          other.constant = otherValue
+        [global]
```

Comment:
The problem with conditions which are not defined is: They will be rendered in Fluid as is: `{$my.constant}` instead of just empty value. That's why I prefer using this updated example and in that case nothing is `ugly` anymore:

```suggestion
        my.constant = my global value
        [traverse(site('configuration'), 'settings/some/setting') == 'someValue']
          my.constant = another value, if condition matches
        [global]
```